MemberDetailsFrontEnd.php

```diff
@@ -0,0 +1,660 @@
+<?php
+
+$connect = new PDO("mysql:host=localhost; dbname=scout;", "root", "");
+
+$query = "SELECT * FROM sections ORDER BY SectionID ASC";
+$statement = $connect->prepare($query);
+$statement->execute();
+$result = $statement->fetchALL();
+
+// function fill_select_box($connect)
+// { 
+//  $output = '';
+//  $query = "SELECT * FROM category ORDER BY CategoryName ASC";
+//  $statement = $connect->prepare($query);
+//  $statement->execute();
+//  $result = $statement->fetchAll();
+//  foreach($result as $row)
+//  {
+//   $output .= '<option value="'.$row["CategoryID"].'">'.$row["CategoryName"].'</option>';
+//  }
+//  return $output;
+// }
+?>
+<html>
+<head>
+  <title>test</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+  <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script> -->
+  <script src="https://rawgit.com/RobinHerbots/jquery.inputmask/3.x/dist/jquery.inputmask.bundle.js"></script>
+
+  <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.14.0/jquery.validate.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/parsley.js/2.8.0/parsley.js"></script> -->
+  <script src="http://parsleyjs.org/dist/parsley.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/parsley.js/2.9.1/parsley.min.js"></script>
+
+  <!-- <script src="https://cdn.datatables.net/1.10.12/js/jquery.dataTables.min.js"></script>
+  <script src="https://cdn.datatables.net/1.10.12/js/dataTables.bootstrap.min.js"></script>   -->
+  <!-- <link rel="stylesheet" href="https://cdn.datatables.net/1.10.12/css/dataTables.bootstrap.min.css" /> -->
+
+  <script src="https://cdn.datatables.net/1.10.22/js/jquery.dataTables.min.js"></script>
+  <script src="https://cdn.datatables.net/1.10.22/js/dataTables.bootstrap4.min.js"></script> 
+  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.22/css/dataTables.bootstrap4.min.css" />
+
+  
+
+  <!-- <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" /> -->
+
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" />
+
+  <!-- <link href="css/bootstrap-select.min.css" rel="stylesheet" />
+  <script src="js/bootstrap-select.min.js"></script> -->
+  
+ 
+ </head>

```

Comment:
The head section should be the same across all pages. You should include a header.php file from all your PHP scripts and put all this stuff inside there.